bids2nda/main.py

```diff
@@ -98,6 +98,9 @@ def cosine_to_orientation(iop):
     express the direction you move, in the DPCS, as you move from row to row,
     and therefore as the row index changes.
 
+    Notes: 
+        Modified Yarik's original solution from https://stackoverflow.com/a/45469577 to use the argmax for increaesd flexibility.
```

Comment:
hm -- was it mine? says user6867490 there. note the typo

```suggestion
        Modified Yarik's original solution from https://stackoverflow.com/a/45469577 to use the argmax for increased flexibility.
```
I will bolt on codespell now

Reply:
actually -- I don't really see a reason for this note here at all.  But I feel it is time to add tests for this function to ensure that it operates as expected... ATM can't quite grasp the ramification of the change, in particular for those cases where before a RuntimeError would have been raised.